modules/tinymce/src/plugins/template/test/ts/browser/DatesTest.ts

```diff
@@ -1,99 +1,95 @@
-import { Log, Logger, Mouse, Pipeline, Step, UiFinder, Waiter } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
-import { TinyApis, TinyLoader } from '@ephox/mcagar';
-import { SugarElement } from '@ephox/sugar';
-import TemplatePlugin from 'tinymce/plugins/template/Plugin';
-import SilverTheme from 'tinymce/themes/silver/Theme';
+import { afterEach, describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks } from '@ephox/mcagar';
 
-UnitTest.asynctest('browser.tinymce.plugins.template.DatesTest', (success, failure) => {
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/template/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
 
-  TemplatePlugin();
-  SilverTheme();
+import { insertTemplate } from '../module/InsertTemplate';
+import { Settings } from '../module/Settings';
 
-  TinyLoader.setupLight((editor, onSuccess, onFailure) => {
-    const tinyApis = TinyApis(editor);
+describe('browser.tinymce.plugins.template.DatesTest', () => {
+  const hook = TinyHooks.bddSetup<Editor>({
+    plugins: 'template',
+    toolbar: 'template',
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Plugin, Theme ]);
 
-    const docBody = SugarElement.fromDom(document.body);
-    const dialogSelector = 'div.tox-dialog';
-    const toolbarButtonSelector = '[role="toolbar"] button[aria-label="Insert template"]';
+  const { addSettings, delSettings } = Settings(hook);
 
-    const sDeleteSetting = (key) => Logger.t('Deleting Setting ' + key, Step.sync(() => {
-      delete editor.settings[key];
-    }));
+  afterEach(() => {
+    const editor = hook.editor();
+    delSettings();
+    editor.setContent('');
+  });
 
-    Pipeline.async({}, [
-      Log.stepsAsStep('TBA', 'Template: Test cdate in snippet with default class', [
-        tinyApis.sSetSetting('templates', [{ title: 'a', description: 'b', content: '<p class="cdate">x</p>' }]),
-        tinyApis.sSetSetting('template_cdate_format', 'fake date'),
-        Mouse.sClickOn(SugarElement.fromDom(editor.getContainer()), toolbarButtonSelector),
-        UiFinder.sWaitForVisible('Waited for dialog to be visible', docBody, dialogSelector),
-        Mouse.sClickOn(docBody, 'button.tox-button:contains(Save)'),
-        Waiter.sTryUntil('Dialog should close', UiFinder.sNotExists(docBody, dialogSelector)),
-        tinyApis.sAssertContent('<p class="cdate">fake date</p>'),
-        tinyApis.sSetContent('')
-      ]),
+  it('TBA: Template: Test cdate in snippet with default class', async () => {
```

Comment:
```suggestion
  it('TBA: Test cdate in snippet with default class', async () => {
```

We need to remove the plugin name from these `it` calls. I've done this above, but there's many others that need cleaning up.

- [ ] Remove template name prefix in tests